packages/storefront-sdk-plugins/commerce-queries/src/graphql/fragments/content.ts

```diff
@@ -0,0 +1,17 @@
+export default /* GraphQL */ `
+	fragment content_content on Content {
```

Comment:
Recommend sticking to the convention of the prefix being the PascalCase GraphQL type:

```suggestion
	fragment Content_content on Content {
```